working-w-copilot/copilot-in-projects-new-code.md

```diff
@@ -88,59 +88,45 @@ These functions aren't very long, but it's still a good idea to test them as a b
 In our `book.py` file, highlight the whole text, bring up the inline Copilot chat with `⌘I` (`CMD + i`), then type in the shortcut `/tests`.
 
 ![VS Code showing the full book.py file contents highlighted with the Copilot inline chat bar showing and "/tests" typed in](assets/copilot-in-projects/book-slash-tests-start.png)  
-*Fig. Selected text in* `book.py` *with the Copilot inline chat up to enter "/tests"*
+*Fig. Selected text in* `book.py` *with the Copilot inline chat up to enter "/tests" ([Full size image](assets/copilot-in-projects/book-slash-tests-start.png))*
 
-Once we hit `Enter`, Copilot will create a temporary section on the VS Code window with unit tests that we can review.
+Once we hit `Enter`, Copilot will add a new pane in the VS Code window with our copilot chat at the top and a temporary file with unit tests that we can review.
 
-![VS Code showing the book.py class and a temporary window where Copilot is previewing unit tests](assets/copilot-in-projects/book-slash-tests-suggestion.png)  
-*Fig. Copilot's UI to preview tests for the* `book.py` *file*
+![VS Code showing the book.py class and a temporary file where Copilot is previewing unit tests](assets/copilot-in-projects/book-slash-tests-suggestion.png)  
+*Fig. Copilot's UI to preview tests for the* `book.py` *file ([Full size image](assets/copilot-in-projects/book-slash-tests-suggestion.png))*
 
-If we feel like the tests presented are a good starting place, we can use the "Create" button to make a new file with those tests pre-populated.
-
-### !callout-warning
-
-## Test File Location
-
-The "Create" button will generate the new test file in the current folder, in our case, in the same folder as our `Book` model. This isn't really what we want, but we can click the arrow next to the "Create" button to see a drop down with a "Create As" option. This will let us change the default name of the file and choose where it gets saved so that we can keep all of our test files together, separate from our source code. If we don't use "Create As", we can drag and drop test files from where they were created to the test folder at any later time.
-
-<br />
-
-![Close up of the Copilot window for previewing tests showing the options drop down from the "Create" button](assets/copilot-in-projects/book-slash-tests-create-as.png)  
-*Fig. "Create As" in the drop down next to the "Create" button*
-
-### !end-callout
+If we feel like the tests presented are a good starting place, we can click "Accept" in the copilot chat to dismiss the chat window, then use `⌘S` (`CMD + S`) to save the new test file. We should see a pop up that allows us to choose the folder where we want to save the test file.
```

Comment:
Sounds good, I can add a few sentences on saving the file in the proper structure to hopefully help folks no get caught up on it.

Reply:
Since we won't be sticking around in this branch beyond this lesson, leaving it where it is might even be fine as long as we have that little nudge. But as a new reader, I don't know what the longevity of this code is going to be, so I think that's where my inclination to want to put it in the "right" place was coming from. So if we don't want fols to get caught up here, maybe even just saying that it's OK to leave it in the same folder since we're just using this as a temporary example could be enough. So whichever makes the most sense to you seems fine.